app/javascript/components/Results/ResultGrid/ResultTitle.tsx

```diff
@@ -2,15 +2,16 @@ import React, { ReactNode } from 'react';
 
 interface ResultTitleProps {
   url: string;
+  className?: string;
```

Comment:
Pulling in @yogesh27 's changes here: https://github.com/GSA/search-gov/commit/13db997834c637a7adf5cd0a45643fca1e13f868